doc/release-notes/5.12-release-notes.md

```diff
@@ -0,0 +1,237 @@
+
+In the following commands we assume that Payara 5 is installed in `/usr/local/payara5`. If not, adjust as needed.
+
+
+# Update to Payara 5.2022.3 highly recommended
+
+With lots of bug and security fixes included, we encourage everyone to update to Payara 5.2022.3 as soon as possible.
+
+**Note:** with the approaching EOL for the Payara 5 Community release train it's likely we will switch to a
+yet-to-be-released Payara 6 in the not-so-far-away future.
+
+We recommend you ensure you followed all update instructions from the past releases regarding Payara.
+(latest Payara update was for [v5.6](https://github.com/IQSS/dataverse/releases/tag/v5.6))
+
+Upgrading requires a maintenance window and downtime. Please plan ahead, create backups of your database, etc.
+
+The steps below are a simple matter of reusing your existing domain directory with the new distribution.
+But we also recommend that you review the Payara upgrade instructions as it could be helpful during any troubleshooting:
+[Payara Release Notes](https://docs.payara.fish/community/docs/Release%20Notes/Release%20Notes%205.2022.3.html)
+
+
+Please note that the deletion of the `lib/databases` directory below is only required once, for this upgrade (see Issue #8230 for details).
+
+```shell
+export PAYARA=/usr/local/payara5
+```
+
+(or `setenv PAYARA /usr/local/payara5` if you are using a `csh`-like shell)
+
+1\. Undeploy the previous version
+
+```shell
+    $PAYARA/bin/asadmin list-applications
+    $PAYARA/bin/asadmin undeploy dataverse<-version>
+```
+
+2\. Stop Payara
+
+```shell
+    service payara stop
+    rm -rf $PAYARA/glassfish/domains/domain1/generated
+    rm -rf $PAYARA/glassfish/domains/domain1/osgi-cache
+    rm -rf $PAYARA/glassfish/domains/domain1/lib/databases
+```
+
+3\. Move the current Payara directory out of the way
+
+```shell
+    mv $PAYARA $PAYARA.MOVED
+```
+
+4\. Download the new Payara version (5.2022.3), and unzip it in its place
+
+5\. Replace the brand new payara/glassfish/domains/domain1 with your old, preserved domain1
+
+6\. Start Payara
+
+```shell
+    service payara start
+```
+
+7\. Deploy this version.
+
+```shell
+    $PAYARA/bin/asadmin deploy dataverse-5.12.war
+```
+
+8\. Restart payara
+
+```shell
+    service payara stop
+    service payara start
+```
+### Additional Upgrade Steps
+
+Update the Citation metadata block:
+
+- `wget https://github.com/IQSS/dataverse/releases/download/v5.12/citation.tsv`
+- `curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @citation.tsv -H "Content-type: text/tab-separated-values"`
+
+- Re-export metadata files (OAI_ORE is affected by the PRs in these release notes). Optionally, for those using the Dataverse software's BagIt-based archiving, re-archive dataset versions archived using prior versions of the Dataverse software. This will be recommended/required in a future release.
+
+- Run ReExportall to update Exports
```

Comment:
This is very minor, but this last line is kind of redundant. Maybe it could be combined with the one above, like "Run ReExportall to update metadata exports (OAI_ORE is affected ... etc."

Reply:
Oh, and if we are mentioning OAI_ORE here, I guess it makes sense to also explicitly mention that the JSON format needs to be re-exported, because of the bug fixed in #8868. 
I'll go ahead and make this change.